examples/create-account.js

```diff
@@ -32,4 +33,36 @@ async function main() {
     console.log(`account = ${newAccountId}`);
 }
 
-main();
+automaticSign();
```

Comment:
the manual sign should be a _new_ example .. perhaps:

 * create-account-with-manual-signing